package.json

```diff
@@ -35,7 +35,7 @@
     "test:classic": "cross-env USE_CLASSIC_JSX=true yarn _test",
     "_test": "jest --config=jest.config.js",
     "test:watch": "yarn test:automatic --watch",
-    "test:ci": "yarn test --coverage && coveralls < coverage/lcov.info",
```

Comment:
`coveralls` was no longer installed - see https://github.com/welldone-software/why-did-you-render/pull/306/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L57